grpc/src/error.rs

```diff
@@ -79,6 +79,39 @@ pub enum Error {
     /// Signing error
     #[error(transparent)]
     SigningError(#[from] SignatureError),
+
+    /// No account
+    #[error("no account")]
+    NoAccount,
+
+    /// Client was not constructed with a signer
+    #[error("Client was not constructed with a signer")]
+    MissingSinger,
+}
+
+/// Representation of all the errors that can occur when building [`GrpcClient`] using
+/// [`GrpcClientBuilder`]
+///
+/// [`GrpcClient`]: crate::GrpcClient
+/// [`GrpcClientBuilder`]: crate::GrpcClientBuilder
+#[derive(thiserror::Error, Debug)]
+pub enum GrpcClientBuilderError {
+    /// Error from tonic transport
+    #[error(transparent)]
+    #[cfg(not(target_arch = "wasm32"))]
```

Comment:
@oblique do you think it's okay to have this cfg in public enum? If some lib uses that, they'd need to also include it, no?

Reply:
Yes they need to include it